### PR TITLE
relative path for restapi | cssclass attribute fix | fix to order

### PR DESCRIPTION
**Pull request by gusreiber**

### CODE REVIEW

File: core/src/main/resources/hudson/model/Job/configure.jelly [hunk 1]
```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} Config" cssClass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
+  <l:layout title="${it.displayName} Config" cssclass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
     <f:breadcrumb-config-outline />
     <l:main-panel>
```
Comment (recena) [inline core/src/main/resources/hudson/model/Job/configure.jelly:32]: This was [changed](https://github.com/recena/jenkins/blob/JENKINS-31162/core/src/main/resources/hudson/model/Job/configure.jelly#L30). Please, upgrade your repo.

File: war/src/main/js/add-item.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ var $ = require('jquery-detached').getJQuery();
 
 var getItems = function(root){
   var d = $.Deferred();
-  $.get(root+'/categories?depth=3').done(
```
Comment (jglick) [inline war/src/main/js/add-item.js:6]: So then the `root` function parameter is unused and should be deleted

Comment (gusreiber) [inline war/src/main/js/add-item.js:6]: yup. change coming.

File: war/src/main/js/add-item.js [hunk 3]
```diff
@@ -106,24 +106,11 @@ $.when(getItems(jRoot)).done(function(data){
...
-                $name.addClass('no-val').focus();
-                missingText.push('item name');
-              }
-              if(!checkItems().selected){
-                $form.addClass('no-select');
-                missingText.push('item type selection');
-              }
-              alert('Required selections are missing: '+ (missingText.join(', ')));
-            });
+          $('<div class="coverz" />').insertAfter($subBtn);
```
Comment (recena) [inline war/src/main/js/add-item.js]: @gusreiber Where this CSS class is defined?

**Pull request merged**